BenadrylAutomation/src/main/java/com/jnj/pageobjects/FooterPage.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * @author Rashi Tiwari
+ * @date 18-March-24
+ */
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class FooterPage extends BaseClass {
+	Actions actions;
+
+	/**
+	 * Constructor
+	 */
+	public FooterPage() {
+		PageFactory.initElements(driver, this);
+		actions = new Actions(driver);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(xpath = "//*[@id='footer']")
+	WebElement footerbanner;
+
+	public WebElement getfooteroption(String link) {
+		return driver.findElement(By.xpath("//a[normalize-space()='" + link + "']"));
+	}
+	
+	public WebElement getSecondfooteroption(String link) {
```

Comment:
@RashiTiwari21 please change it to "getSecondFooterOption"

Reply:
fixed

Reply:
Verified @RashiTiwari21